source/docs/stats.md

```diff
@@ -262,3 +262,30 @@ Developers trying to can iterate through changes in these tests locally with:
       test/integration:stats_integration_test
 ```
 
+## Debugging Symbol Table Assertions
+
+If you are visiting this section because you saw a message like:
+
+```bash
+[2020-12-11 18:30:54.059][16][critical][assert] [source/common/stats/symbol_table_impl.cc:251] assert failure: decode_search != decode_map_.end(). Details: Please see https://github.com/envoyproxy/envoy/blob/master/source/docs/stats.md#debugging-symbol-table-asserts for hints on how to fix
```

Comment:
Removing the timestamp will require a bit less horizontal scrolling on the rendered page.

Reply:
Good catch; actually I'll go ahead and add some newlines for doc purposes.